com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -169,19 +168,6 @@ public void HandleAddObject(ulong clientId, Stream stream)
                 var networkObject = NetworkManager.SpawnManager.CreateLocalNetworkObject(softSync, prefabHash, ownerClientId, parentNetworkId, pos, rot, isReparented);
                 networkObject.SetNetworkParenting(isReparented, latestParent);
                 NetworkManager.SpawnManager.SpawnNetworkObjectLocally(networkObject, networkId, softSync, isPlayerObject, ownerClientId, stream, hasPayload, payLoadLength, true, false);
-
```

Comment:
Yay!
This fixes the anoying race condition in #729 🎉

Reply:
🎊